login-workflow/example3/src/actions/AuthUIActions.tsx

```diff
@@ -50,14 +51,16 @@ export const ProjectAuthUIActions: AuthUIActionsWithSecurity = (securityHelper)
         // screen will be unmounted and thrown away.
         // securityHelper.onUserAuthenticated()
         if (authData?.email !== undefined) {
-            securityHelper.onUserAuthenticated({
-                email: authData?.email,
-                userId: authData.userId ?? '',
-                rememberMe: authData?.rememberMeData.rememberMe,
-            });
+            // securityHelper.onUserAuthenticated({
+            //     email: authData?.email,
+            //     userId: authData.userId ?? '',
+            //     rememberMe: authData?.rememberMeData.rememberMe,
+            // });
+            appHelper.setIsAuthenticated(true);
```

Comment:
Is the `onUserAuthenticated` functionality/data handled elsewhere? If so, let's get rid of these comments

Reply:
Created BLUI-4499 in the backlog for onUserAuthenticated.